finality-aleph/src/validator_network/manager/mod.rs

```diff
@@ -48,7 +59,7 @@ impl ManagerStatus {
         let incoming: HashSet<_> = manager
             .incoming
             .iter()
-            .filter(|(_, exit)| !exit.is_canceled())
+            .filter(|(_, exit)| !exit.is_closed())
```

Comment:
why this change?

Reply:
oh, because of the change from oneshot to unbounded, right? then why this one?

Reply:
See comment above.